### PR TITLE
Added search docs.

### PR DESCRIPTION
**Pull request by blackbaud-brandonhare**

### CODE REVIEW

File: src/app/content/search/index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<stache
+  pageTitle="Search">
+
+  <p>
+    Stache has a search feature that enables users to quickly find content. This feature is only available to Blackbaud employees. To learn more, check out our <a href="https://blackbaud.visualstudio.com/Products/User%20Ed%20-%20SDK%20API/_git/stache-search-query-svc?path=%2FREADME.md&version=GBmaster&_a=preview">internal docs</a>.
```
Comment (blackbaud-johnly) [inline src/app/content/search/index.html]: A few minor suggestions:

In the first sentence, I'd change "has" to "includes."

I'd change the second sentence to "This feature is for internal Blackbaud use only." This is the wording that we use on the SKY UX site. There's nothing wrong with your version. I'm just suggesting this version for consistency.

I'd tweak the the third sentence to make it more explicit that the docs are only for Blackbaud users. This is implicit by our use of "internal," but I think starting with the generic "To learn more" is potentially confusing and/or jarring. I'd change the sentence to "For Blackbaud developers, <a href="https://blackbaud.visualstudio.com/Products/User%20Ed%20-%20SDK%20API/_git/stache-search-query-svc?path=%2FREADME.md&version=GBmaster&_a=preview">the Stache Search Query Service README</a> describes search capabilities for Stache sites."

Comment (Blackbaud-DeniseCisneros) [reply src/app/content/search/index.html]: I agree with John's comments! Should we use some sort of specific styling for the internal use only comments or is that maybe overkill?

File: src/app/content/demo-content.service.ts [hunk 2]
```diff
@@ -151,6 +151,12 @@ export class StacheDemoContentService {
       route: '/content/affix-top',
       icon: 'wrench',
       summary: `Wraps elements and causes them to affix to the window on scroll.`
+    },
+    {
+      name: 'Search',
+      route: '/content/search',
+      icon: 'search',
+      summary: `Allows site content to be searchable.`
```
Comment (blackbaud-johnly) [inline src/app/content/demo-content.service.ts]: Fine as is, but what do you think of "Enables search capabilities for site content"? I prefer that it uses a more active voice., but I don't feel that strongly about it.

File: src/app/content/search/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/content/search/index.html]: We don't use any styling on SKY UX for this sort of thing. I can see where a specific styling might make sense, but I suspect we don't want to draw any more attention to these internal-only areas than we already do.

**Comment by blackbaud-brandonhare** [comment]
Anyone have any objections to me merging this?

**Pull request merged**